.gitattributes

```diff
@@ -0,0 +1,19 @@
+# See https://git-scm.com/docs/gitattributes
+# See https://help.github.com/articles/dealing-with-line-endings/
+
+# Default behavior, if core.autocrlf is unset.
+* text=auto
+
+# Files to be converted to native line endings on checkout.
+*.cpp text
+*.h text
+
+# Public header files will always have LF (unix) line endings on checkout.
+vk_*.h text eol=lf
+vk_*.hpp text eol=lf
```

Comment:
Verified this fixes my issue on Windows when running `python scripts/generate_source.py external/Debug/Vulkan-Headers/registry/`